odoo_backup_sh/doc/index.rst

```diff
@@ -25,19 +25,40 @@ Configuration
   server_wide_modules = web,odoo_backup_sh
   # (...)
 
+In order for desired databases to appear in a backup dashboard, you have to create settings for it.
+Open the menu ``[[ {Backups} ]] >> {Databases} >> {Databases}``. Click on the ``[{Create}]`` button.
```

Comment:
Don't use extra brackets. It shoudl be just ``[[ Backups ]]``. Check formating guidelines

Reply:
Why don't create New Settings just from Dashboard?